modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
+"loggingRules": {
+      "agents.partners": {
+        "bucket": "conflictlogsbucket2",
+        "collection": "myappscope.partnerlogs"
+      },
+      "agents.retailers": null,
+      "inventory": {
+        "bucket": "conflictlogsbucket2",
+        "collection": "myappscope.inventorylogs"
+      }
+    }
+}'
+----
+
+[#disabled-with-stored-logging-rules]
+==== Disabled with stored logging rules
+
+You can configure `conflictLogging` (specify the default conflict collection and the `loggingRules`) but not enable `conflictLogging` by setting `disabled` to `true`. 
+
+NOTE: You can update the replication and set the `conflictLogging` status `disabled` to `true` or `false` at any time.
+
+When you set `"disabled": true`, it also disables conflict logging, but all of the rules are retained for the collection.
+
+*Format:*
+
+[source,json]
+----
+--data-urlencode conflictLogging= '{
+                    "bucket": [bucketName],
+                    "collection": [scope].[collection],
+                    "loggingRules": {[logging_rules]},
+                    "disabled": true,
+}'
+----
+
+*Example:* Conflict logging is disabled, but the default conflict collection and the logging rules are retained.
+
+[source,json]
+----
+--data-urlencode conflictLogging= '{
+                    "disabled": true,
+                    "bucket": "conflictlogsbucket1",
+                    "collection": "myappscope.log1collection",
+                    "loggingRules": {
+                        "agents.partners": {
+                        "bucket": "conflictlogsbucket2",
+                        "collection": "myappscope.partnerlogs"
+                    },
+                    "agents.retailers": null,
+                    "inventory": {
+                        "bucket": "conflictlogsbucket2",
+                        "collection": "myappscope.inventorylogs"
+                    }
+            }
+}'
+----
+
+[#disabled-conflict-logging]
+==== Disabled
+
+To disable conflict logging, set `--data-urlencode conflictLogging='{}'`.
+
+[#logging-rules]
+=== loggingRules
+
+Use `loggingRules` to override the xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#enabled-for-all-conflicts[default conflict logging settings].
+
+[NOTE]
+====
+* The keys and their priorities of each KV pair are specified in the format similar to xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc##rules-for-explicit-mappings[Rules for Explicit Mappings] and also have the same priority as listed in the mapping rules.
+
+* Similar to explicit mapping mode, if a source scope or collection is not specified, then the conflict from that source is ignored.
+
+* Conflict logging rules can be saved without enabling the Conflict Logging setting, however, the rules will come into effect only when the Conflict Logging setting is enabled.
+====
+
+IMPORTANT: If logging fails, XDCR continues with conflict resolution to ensure High Availability.
```

Comment:
Should remove line 426 -- it doesn't belong here.  It looks out of place.
It's important info that should be presented much earlier in the general descriptions about the feature.
I think that that info is already in the general info sections.